docs/Features/XMLFeatures.md

```diff
@@ -0,0 +1,59 @@
+# XML Features
+
+## XML Tag Auto Close
+
+[vscode-xml](https://github.com/redhat-developer/vscode-xml) provides an `auto close` feature for XML tags:
+
+Using `/` in an opening tag will auto close the tag.
+
+![Auto Close Tag Slash](../images/Features/AutoCloseTagSlash.gif)
+
+Using a `>` at the end of an opening tag generates the corresponding closing tag.
+
+![Auto Close Tag End](../images/Features/AutoCloseTagEnd.gif)
+
+## Linked Editing
```

Comment:
Please remove this feature, since it is not merged yet. We will add this section once the feature will be merged.